subliminal/providers/legendastv.py

```diff
@@ -266,12 +275,19 @@ def get_archives(self, title_id, language_code):
                                             'pack' in archive_soup['class'], 'destaque' in archive_soup['class'],
                                             self.server_url + archive_soup.a['href'][1:])
 
+                data_text = archive_soup.find('p', class_='data').text
+
                 # match rating
-                match = rating_re.match(archive_soup.find('p', class_='data').text)
+                match = rating_re.match(data_text)
                 archive.downloads = int(match.group('downloads'))
                 if match.group('rating') is not None:
                     archive.rating = int(match.group('rating'))
 
+                # timestamp
+                match = timestamp_re.search(data_text)
+                if match:
```

Comment:
I don't remember the case right now, but I do remember that exists entries without it (I believe that they are related to old entries that came from a site migration).

And I believe that some timestamps might not represent a valid date and time (e.g.: 13/13/2010 - 11:11), but since I'm not sure about this last case, better to not handle it and if we get a case, then solve it.